src/view/pages/Farmer/tabs/Info/index.tsx

```diff
@@ -1,48 +1,34 @@
 import * as React from 'react';
-import { Container, Header, H1, H2, H3, Content, Grid, Row, Col, Form, Item, Input, Label, Radio, Button, Right, ListItem, InputGroup, Picker }
-from 'native-base';
-import { View, Text, StyleSheet } from 'react-native';
 
 import ViewFarmer from './viewFarmer';
 import EditFarmer from './editFarmer';
 import AddFarmer from './addFarmer';
-
-import styles from './style';
 import createPage from '../../../../generators/Page/index';
 
-interface OwnState {
+interface OwnStateType {
   mode?: string;
 }
 
-export interface OwnProps {
-  // farmerFirstName: string;
-  // farmerLastName: string;
-  // farmerPhoneNumber: string;
-  // farmerBusinessName: string;
-  // farmerNotes: string;
-  // selectedPaymentCycle: string;
-  // selectedPaymentMethod: string;
-  // modeHandler?: Function;
-}
-
+/**
+ * Modes to determine how to render farmer information
+ */
 export const modes = {
-  add: 'add',
   edit: 'edit',
   view: 'view',
 };
 
 /**
  * Container for application
  */
-class FarmerInformation extends React.Component<OwnProps, OwnState> {
+class FarmerInformation extends React.Component<{}, OwnStateType> {
 
   private initialProps: any;
 
   public static defaultProps = {
     mode: 'view',
   };
 
-  constructor(props: OwnProps) {
+  constructor(props: any) {
     super(props);
     this.state = { 
       mode: 'view',
```

Comment:
Maybe it would be a good idea to create a `export type PageMode = 'view' | 'edit'` so the other pages can access this type as I mentioned earlier in the review